app/models/vm_or_template.rb

```diff
@@ -200,6 +201,9 @@ class VmOrTemplate < ApplicationRecord
   delegate :connect_lans, :disconnect_lans, :to => :hardware, :allow_nil => true
   delegate :queue_name_for_ems_operations, :to => :ext_management_system, :allow_nil => true
 
+  supports_attribute :feature => :reconfigure_network_adapters
```

Comment:
While here we might as well add the rest which I believe includes CPU memory and disk.  @GilbertCherrie can give the details.

Reply:
@kbrock the other 3 item_supports? features in ui-classic are:
```:reconfigure_disks```
```:reconfigure_disksize```
```:reconfigure_cdroms```

Reply:
Is there no check for CPU and memory?

Reply:
```        :allowMemoryChange   => role_allows?(:feature => 'vm_reconfigure_memory'),
        :allowCpuChange      => role_allows?(:feature => 'vm_reconfigure_cpu'),
        :allowDiskChange     => role_allows?(:feature => 'vm_reconfigure_disks') && item_supports?(:reconfigure_disks),
        :allowDiskSizeChange => item_supports?(:reconfigure_disksize),
        :allowNetworkChange  => item_supports?(:reconfigure_network_adapters) && role_allows?(:feature => 'vm_reconfigure_networks'),
        :allowCdromsChange   => item_supports?(:reconfigure_cdroms) && role_allows?(:feature => 'vm_reconfigure_drives'),
        :isVmwareInfra       => vm.vendor == 'vmware' && vm.type.include?('InfraManager'),
        :isVmwareCloud       => vm.vendor == 'vmware' && vm.type.include?('CloudManager'),
        :isRedhat            => vm.vendor == 'redhat',
```

@Fryguy This is the whole list. I'm not sure if the role_allows? features are the same as the item_supports? ones or not.